### PR TITLE
category selection with tests

### PR DESCRIPTION
**Pull request by EmilioTyl**
** Summar **
Fix of selection category

### CODE REVIEW

File: app/views/campaigns/new.html.slim [hunk 1]
```diff
@@ -40,14 +18,9 @@
...
-      = f.hidden_field :category, id: 'category-input', required: true
-      - Campaign::CATEGORIES.each do |c|
-        .col-md-4.col-sm-4.col-xs-12.category-card
-          .inner-card data-category=c
-            = image_tag 'cat-blanco.png', class: 'cat-image'
-            .category-name
-              p
-                = t("campaigns.categories.#{c}")
+      .col-sm-5.white-bold
+        = f.collection_select(:category_id, Category.find(:all), :id, :name,{}, {class: 'form-control'})
```
Comment (francoprud) [inline app/views/campaigns/new.html.slim]: Prefer `Category.all` instead of `Category.find(:all)`

**Comment by francoprud** [comment]
@EmilioTyl

**Comment by EmilioTyl** [comment]
@francoprud ya esta

**Comment by francoprud** [comment]
:+1:

**Pull request merged**